app/src/main/res/layout/activity_bill_details.xml

```diff
@@ -1,9 +1,54 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".home.activities.BillDetailsActivity">
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/tvLocationDetails"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="30dp"
+        android:gravity="center"
+        android:hint="@string/restaurant_name"
+        android:textColor="@color/black"
+        android:textSize="30dp"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
+    <TextView
+        android:id="@+id/tvBillAmountDetails"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:gravity="center"
+        android:hint="@string/total_bill_amount"
+        android:textColor="@color/black"
+        android:textSize="23dp"
+        app:layout_constraintTop_toBottomOf="@id/tvLocationDetails" />
+
+
+    <TextView
+        android:id="@+id/tvAmountsOwedDetails"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:gravity="center"
+        android:hint="Tina owes 20"
```

Comment:
Might not be needed? Since this is not an input field. In general, if there's no chance for the hint to appear, we should just leave it out in XML.